### PR TITLE
Add New Game Option to the File Menu

### PR DESCRIPTION
**Pull request by pizza2004**
I added a menu option to bring up the Scenario Selector from within
game, so that you can easily choose a new level without needing to load
the Title Screen.

### CODE REVIEW

File: src/localisation/string_ids.h [hunk 1]
```diff
@@ -2259,6 +2259,8 @@ enum {
 	
 	STR_CMD_PLUS = 5633,
 	STR_OPTION_PLUS = 5634,
+	
+	STR_NEW_GAME =5635,
```
Comment (LinusU) [inline src/localisation/string_ids.h]: Missing space between `=` and `5635`

File: src/windows/top_toolbar.c [hunk 2]
```diff
@@ -425,7 +427,7 @@ static void window_top_toolbar_mousedown(int widgetIndex, rct_window*w, rct_widg
 
 #ifndef DISABLE_TWITCH
 		if (_menuDropdownIncludesTwitch && gTwitchEnable) {
-			dropdown_set_checked(11, true);
+			dropdown_set_checked(13, true);
```
Comment (LinusU) [inline src/windows/top_toolbar.c]: I might be missing something, but why was this value incremented by `2`?

Comment (pizza2004) [inline src/windows/top_toolbar.c]: I'm pretty sure that when they added the Save Game As option they forgot to increment it.

Comment (IntelOrca) [inline src/windows/top_toolbar.c]: separators

File: data/language/english_uk.txt [hunk 3]
```diff
@@ -3972,7 +3972,8 @@ STR_5630    :Enable unlocking of scenarios
 STR_5631    :Original DLC Parks
 STR_5632    :Build your own...
 STR_5633    :CMD + 
-STR_5634    :OPTION + 
+STR_5634    :OPTION +
```
Comment (marijnvdwerf) [inline data/language/english_uk.txt]: You accidentally removed the trailing space

File: src/windows/top_toolbar.c (hunk 2 above)
Comment (pizza2004) [inline src/windows/top_toolbar.c]: Is it wrong then?

**Comment by pizza2004** [comment]
I added the two missing spaces.  Anything else?

**Comment by Gymnasiast** [comment]
Test whether the menus in the Scenario Editor, Roller Coaster Designer and Track Designs Manager still work. (All the menu options in all three mentioned game tools.)

**Comment by pizza2004** [comment]
I fixed the scenario editor menu so it works properly as well, the other two had no issues.

**Comment by wolfreak99** [comment]
i really hope this gets merged. this is really needed.

**Comment by AaronVanGeffen** [comment]
Unfortunately, the PR must be rebased before it can be merged.

**Comment by pizza2004** [comment]
*sighs* Okay I've been trying to do the rebase effectively for like half an hour.  I keep running into weird issues.  I'd love some help.

**Comment by AaronVanGeffen** [comment]
Assuming you have the main OpenRCT2 repo configured as `upstream`, checkout your branch and run:
`git rebase -i upstream/develop`. Applying your commit, you'll get a conflict on the string you added. Amend the files in question and add them with `git add`, then proceed with `git rebase --continue`. You do not need to make a new commit. After the rebase is completed, double-check it compiles and then force-push it using `git push -f`.

**Comment by pizza2004** [comment]
Okay, thanks.  That finally worked.  It should be good to merge now!

**Comment by pizza2004** [comment]
Anything else anyone can find that needs to be changed?

**Comment by IntelOrca** [comment]
If you start a new game on multiplayer, it will stay connected to the server. You need to close the network connection when you play a new scenario.

```c
if (network_get_mode() == NETWORK_MODE_CLIENT) {
    network_close();
}
```

**Comment by pizza2004** [comment]
Alright, I added in the checks for multiplayer.

**Comment by pizza2004** [comment]
I tested this online and it works perfectly in multiplayer.

**Comment by pizza2004** [comment]
I rebased again for good measure.

**Pull request merged**